src/pages/studyView/studyPageHeader/StudyPageHeader.tsx

```diff
@@ -1,25 +1,31 @@
-import * as React from 'react';
-import {observer} from "mobx-react";
-import {StudyViewPageStore} from 'pages/studyView/StudyViewPageStore';
+import * as React from "react";
+import { observer } from "mobx-react";
+import { StudyViewPageStore } from "pages/studyView/StudyViewPageStore";
 import RightPanel from "./rightPanel/RightPanel";
 import StudySummary from "./studySummary/StudySummary";
 import UserSelections from "../UserSelections";
-import * as _ from 'lodash';
-
+import { keys, values } from "lodash";
```

Comment:
I'm not sure about this.  I understand that if we do this, we could potentially save weight of unused lodash methods (if tree shake), but the whole library is only 24k gzipped and I think we probably use more than half of the methods.  The problem above is that it's harder to know the functions are from lodash.  You have to look up to imports if you want to make sure.  Would rather stick with what we have.

Reply:
Reverted. I can see your point, its something I do out of habbit, and as a best practice when writing something from scratch.